charts/redis/Chart.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v2
+name: redis
+description: A Helm chart to deploy the redis for dev and test environment
+type: application
+# HelmChart Version
+version: 1.0.1
```

Comment:
As this is the first version, let it be `1.0.0`

Reply:
Done, It's updated